### ISSUE TITLE
auto breed addition 

### ISSUE THREAD

**Issue by iwilleaturwaffle** [MeteorDevelopment/meteor-client#4545]
### Describe the feature

when I use auto breed I have to keep turning it off and on again, I think a keep on after activate button would be very helpful.

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by abb3v** [comment]
What?
>  a keep on after activate button
Modules are already toggled? Can you elaborate what it is you are talking about?

**Comment by iwilleaturwaffle** [comment]
yes sorry when i turn on the auto breed module it will breed one of each
mob and then stop.
In the video seen below I only
 Screen recorder_04-24-2024_16-47-26-623.mp4
<https://drive.google.com/file/d/1seqLnpiwsJ6TIzSNKYTcVD8YNemWPFRp/view?usp=drive_web>
use baby cows as it's easier to show, but you can see that after the module
feeds the babies once it doesn't turn off but it doesn't keep breeding
either.
sorry about having to download the video the file was too large

**Comment by dudethis** [comment]
I think I understand what you mean, the autobreed module only feeds some of the animals, but not all of them

**Comment by iwilleaturwaffle** [comment]
no sorry if i wasn't clear. the module would breed all the mobs, but only
once.
